### PR TITLE
GRF-135: handle cookies and userinfo requests

### PR DESCRIPTION
**Pull request by val92130**
## 📝 What is this PR

Added handlers to be able to query cookies and "userInfo" from anywhere in the React app.

## 🏭 How does it work

I moved the code that handles React event messages from the "setup handler" to the "phantombuster" handler which means the messages are intercepted from anywhere in the app.

I also added the ability to query cookies by sending a `FETCH_COOKIES` event from the React app.

## 🧪 QA Process

You can add this code snippet to try out the new event:

````typescript
	React.useEffect(() => {
		const handleExtensionMessage = (event: Event) => {
			if ("data" in event) {
				console.log("RECEIVED", event.data)
			}
		}

		const requestCookies = () => {
			const eventPayload = {
				type: "FETCH_COOKIES",
				payload: {
					website: "LinkedIn",
				},
			}
			const event = new CustomEvent("PB_APP_MESSAGE", {
				detail: eventPayload,
			})
			window.dispatchEvent(event)
		}

		window.addEventListener("message", handleExtensionMessage)
		requestCookies()
		return () => window.removeEventListener("message", handleExtensionMessage)
	}, [])
 ````

## 👌 To check

-   [ ] There's no regression
-   [ ] Coding guidelines are followed ([Contributing](https://github.com/phantombuster/engineering-docs/blob/master/contributing.md), [Testing](https://github.com/phantombuster/engineering-docs/blob/master/testing.md))

### CODE REVIEW

File: beta-updates.json [hunk 1]
```diff
@@ -49,6 +49,10 @@
 				{
 					"version": "1.1.9.0",
 					"update_link": "https://github.com/phantombuster/web-browser-extension/releases/download/v1.1.9/firefox-1.1.9.0.xpi"
+				},
+				{
+					"version": "1.1.10.0",
+					"update_link": "https://github.com/phantombuster/web-browser-extension/releases/download/v1.1.9/firefox-1.1.10.0.xpi"
```
Comment (GabrielCousin) [inline beta-updates.json:55]:
```suggestion
					"update_link": "https://github.com/phantombuster/web-browser-extension/releases/download/v1.1.10/firefox-1.1.10.0.xpi"
```

**Review by GabrielCousin — COMMENTED** [review]
Looks fantastic! Tested with Chrome and Firefox 🦊

**Review by RiahiKarim — APPROVED** [review]
LGTM 💯  it works great!

**Review by Hatscat — DISMISSED** [review]
nice! LGTM 💯

**Review by GabrielCousin — APPROVED** [review]

**Review by GabrielCousin — APPROVED** [review]

**Review by Hatscat — APPROVED** [review]

**Review by RiahiKarim — APPROVED** [review]

**Pull request merged**